modular_skyrat/code/game/objects/items.dm

```diff
@@ -12,3 +16,39 @@
 		user.visible_message("<span class='danger'>[t]</span>")
 	if(grabsound)
 		playsound(src,grabsound,50,1)
+
+//TGUI item info menu
+/obj/item/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, \
```

Comment:
/obj/item owning this UI is very bad due to inheritance stuff, any other /obj/item that has a TGUI part will fuck up, such as oxygen tanks. You'll have to put the UI under a datum and set ui_host to the item. Not hard.

Reply:
going to make an element